### DISCUSSION TITLE
adc mcrophone max9814 result

### DISCUSSION THREAD

**Discussion by aqildad-create** [pschatzmann/arduino-audio-tools#730] [General]
test max9814 microphone and run adc to seral sketch , below is plotter result with screaming and without  screaming ? 

Is this normal in term of amplitude ? 

without screaming 
![image](https://user-images.githubusercontent.com/65456998/228769145-3100bb0c-e40b-4533-b1ed-d8c4c4080b2f.png)

with screaming 
![image](https://user-images.githubusercontent.com/65456998/228769358-c2166281-7acb-4533-a033-ad5427a4faa4.png)

**Comment by pschatzmann** [comment]
The signal should be centered around 0. The max volume range of int16_t is -32,768 .. 32,767.
I think values below the thousands are barely audible.

I would suggest to use a [generated sine tone](https://onlinetonegenerator.com/) for testing. This should make it easier to verify the result...
